### PR TITLE
Feature/update hotel test case

### PR DESCRIPTION
**Pull request by sathish199308**
Update Test Case for Rooms and Hotels. XML Comments also implement

### CODE REVIEW

File: BookingSystem.Tests/BAL.Tests/HotelServiceTest/UpdateHotelTest.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        public UpdateHotelTest()
+        {
+            _hotelRepository = new Mock<IHotelRepository>();
+            fixture = new Fixture();
+        }
+
+        [Fact]
+        public async Task UpdateHotel_ReturnOKResult_WithHotelId()
+        {
+            // Arrange
```
Comment (Backiaraj07) [inline BookingSystem.Tests/BAL.Tests/HotelServiceTest/UpdateHotelTest.cs:29]: Please add summary, and give a method name or give correct name in summary remarks

File: BookingSystem.Tests/BAL.Tests/RoomServiceTest/UpdateRoomTest.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            _roomRepository = new Mock<IRoomRepository>();
+            _rateCardRepository = new Mock<IRateCardRepository>();
+            fixture = new Fixture();
+        }
+
+        #region Update Room OK Result Test Case
+
+        /// <summary>
+        /// Update Room OK Result Test Case
+        /// </summary>
```
Comment (Backiaraj07) [inline BookingSystem.Tests/BAL.Tests/RoomServiceTest/UpdateRoomTest.cs:32]: use proper comments

**Review by Backiaraj07 — COMMENTED** [review]
comments added

File: BookingSystem.Tests/BAL.Tests/HotelServiceTest/UpdateHotelTest.cs (hunk 1 above)
Comment (sathish199308) [reply BookingSystem.Tests/BAL.Tests/HotelServiceTest/UpdateHotelTest.cs:29]: I added Summary, Kindly check

File: BookingSystem.Tests/BAL.Tests/RoomServiceTest/UpdateRoomTest.cs (hunk 2 above)
Comment (sathish199308) [reply BookingSystem.Tests/BAL.Tests/RoomServiceTest/UpdateRoomTest.cs:32]: I added the summary, kindly check

**Review by Backiaraj07 — APPROVED** [review]
Comments fixed, verified Code.

**Review by manic1990 — APPROVED** [review]

**Pull request merged**